### DISCUSSION TITLE
Looking forward to the next release with DW v2.0.27 :-)

### DISCUSSION THREAD

**Discussion by siwonred** [xvik/dropwizard-guicey#203] [General]
The Dropwizard v2.0.27 was released just now 🎉
Our team is looking forward to the next release of this project 🙏

**Comment by xvik** [comment]
Ok, I believe many people would ask for a new release this time. I'll do a quick fix for #187 and release a new version with it this weekend.

**Comment by xvik** [comment]
[5.4.1 released](https://github.com/xvik/dropwizard-guicey/releases/tag/5.4.1)
